src/main/java/ttt/gui/TicTacToeRules.java

```diff
@@ -72,10 +73,10 @@ private void togglePlayer() {
     }
 
 
-   @Override
-   public boolean gameInProgress() {
-      return boardHasFreeSpace() && !hasWinner();
-   }
+    @Override
+    public boolean gameInProgress() {
+        return boardHasFreeSpace() && !hasWinner();
```

Comment:
I just realised this (_I know this code has been there for a while_). Generally I like code that is a little more expressive. So, for example, in this line you say:

`!hasWinner()`; basically saying does not have a winner. Consider creating a method that expresses this concept instead of denying (_reversing_) the meaning of the returned `boolean` value. Writing something like:

`return boardHasFreeSpace() && noWinnerYet()` reads almost like a sentence and you will not miss the *!* (which can be overseen)